### ISSUE TITLE
GitHub `RepoStats` Broken

### ISSUE THREAD

**Issue by david-poindexter** [DNNCommunity/DNNDocs#692]
## Description of bug
It looks like the GitHub `RepoStats` are no longer working to populate `Top Contributors` and `Recent Contributors` on the home page.

## Steps to reproduce
List the steps to reproduce the behavior:
1. Go to https://docs.dnncommunity.org
2. Look in top right area of the home page
3. See `Top Contributors` are missing images and stats
4. See `Recent Contributors` are no longer listed

## Current result
See screenshot below.

## Expected result
This data should be pulled from the GitHub API at build time to generate the necessary sections of the static site.

## Screenshots
![image](https://user-images.githubusercontent.com/4568451/214113873-b2dad10d-186a-4478-b137-8820921ab593.png)

## Error
Unknown.

## Additional context
This is related to a custom `DocFx` plugin for RepoStats, but most like is related to the GitHubApi provider within the custom plugin project.  My suspicion is it is related somehow to the GitHub token, but this will require further investigation.

## Affected browser
* [x] Chrome
* [x] Firefox
* [x] Safari
* [x] Edge

**Comment by valadas** [comment]
So the top links in each page is now working, as far as I know top contributors too but I can't see it rendered, I think this was hidden in css maybe due to it being broken ?

**Comment by valadas** [comment]
Yep, I ran out of time but the html is commented out, so must just need to be uncommented in our template

![Image](https://github.com/user-attachments/assets/7cc158dc-83ff-451d-9355-c9297650c01c)

**Comment by david-poindexter** [comment]
I am working on a PR for this now.
